packages/ldes-mongodb-connector/lib/MongoDBConnector.ts

```diff
@@ -0,0 +1,89 @@
+import type { IWritableConnector } from '@ldes/types';
+// import mongoose, { Schema } from 'mongoose';
+import { MongoClient } from 'mongodb';
+
+export class MongoDBConnector implements IWritableConnector {
+  private readonly config: any;
+  private db: any;
+
+  public constructor(config: any) {
+    this.config = config;
+  }
+
+  /**
+   * Writes a version to the corresponding backend system
+   * @param member
+   */
+  public async writeVersion(member: any): Promise<void> {
+    var JSONmember;
+    try {
+      JSONmember = JSON.parse(member);
+    } catch {
+      console.log(member);
+    }
+
+    const collection = this.db.collection('ldes');
+
+    let id = JSONmember['@id'];
+    let type = JSONmember['@type'];
+    let time = JSONmember['http://www.w3.org/ns/prov#generatedAtTime'];
+    let isVersionOf = JSONmember['http://purl.org/dc/terms/isVersionOf']['@id'];
```

Comment:
This should be part of a generic solution to create a version materialization of an LDES. dcterms:isVersionOf is just one way of having a link to a higher level version thing. We could for example also think about `all the latest air quality observations sosa:madeBy a sensor`.

 I think the first focus should be to store _all_ the version objects as they are, and not yet do the version materialization, which probably should be a feature of the LDES client itself.

Related issue: https://github.com/TREEcg/event-stream-client/issues/13

Reply:
Absolutely, we have to look at this!

I think that this storage takes part in the discovery of what is possible.
Storing isVersionOf allows us to have a granularity in what is kept or not (number of version to keep, etc.)

Reply:
I created an issue to keep track of this: https://github.com/osoc21/ldes2service/issues/20